run-tests-with-build-kogito-examples-parent/kogito-examples-project-sources-test/pom.xml

```diff
@@ -138,6 +138,11 @@
                   <quarkus.native.container-build>${quarkus.native.container-build}</quarkus.native.container-build>
                   <quarkus.native.container-runtime>${quarkus.native.container-runtime}</quarkus.native.container-runtime>
                   <quarkus.native.builder-image>${quarkus.native.builder-image}</quarkus.native.builder-image>
+                  <pomExcludes>
```

Comment:
Wrong nesting, this should be outside of `<properties>`

Reply:
Thanks for noticing, fixed